azext_iot/iothub/providers/job.py

```diff
@@ -128,37 +132,37 @@ def create(
             in [JobType.scheduleUpdateTwin.value, JobType.scheduleDeviceMethod.value]
             and not query_condition
         ):
-            raise CLIError(
+            raise RequiredArgumentMissingError(
                 "The query condition is required when job type is {} or {}. "
                 "Use query condition '*' if you need to run job on all devices.".format(
                     JobType.scheduleUpdateTwin.value, JobType.scheduleDeviceMethod.value
                 )
             )
 
         if poll_duration < 1:
-            raise CLIError("--poll-duration must be greater than 0!")
+            raise InvalidArgumentValueError("--poll-duration must be greater than 0!")
```

Comment:
Why is there an ! and not a . ?

Reply:
The string content wasn't changed in this PR.

Reply:
Just a general why - I do not expect a string change here but it does feel weird to have an !

Reply:
Probably some small micro-decision that happened over 2 years ago.

Reply:
nit: can we change this so it is less awk

Reply:
Feel free to change it to period if you desire.